clients/drcachesim/tools/view.cpp

```diff
@@ -278,6 +291,7 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             // since memref iterators use the timestamps to order buffer units.
             timestamp_ = memref.marker.marker_value;
             timestamp_record_ord_ = memstream->get_record_ordinal();
+            timestamp_tid_ = memref.marker.tid;
```

Comment:
There's also the pid that would need to be saved.

Maybe just store the whole memref instead of reconstructing it in parallel_shard_exit?

Reply:
Done.